r/NEWS.md

```diff
@@ -19,6 +19,17 @@
 
 # arrow 14.0.2.9000
 
+##  Minor improvements and fixes
+
+* Throw error when `string_replace` is passed vector of values in `pattern` (#39219)
```

Comment:
```suggestion
* Throw error when the binding for `stringr::string_replace()` is passed vector of values in `pattern` (#39219)
```

Reply:
```
> stringr::string_replace
Error: 'string_replace' is not an exported object from 'namespace:stringr'
```

Isn't is `str_replace_all()`?

Reply:
It seems it's for the family of `sub` type functions https://github.com/apache/arrow/pull/39219/files#diff-e992712511690afeafbd88380b92e2ffb2f06453f6b9122afda7273a44a4aca4R351